cypress/support/commands/form.js

```diff
@@ -381,23 +370,23 @@ Cypress.Commands.add('selectNthInListField', (fieldName, index, modal) => {
 Cypress.Commands.add('setCheckBoxValue', (fieldName, isChecked, modal = false, rewriteUrl = null, skipRequest = false) => {
   cy.log(`Set the Checkbox value ${fieldName} to ${isChecked}`);
 
-    // the expected value is the same as the checked state
-    // (used only for verification if the checkbox has the correct value)
-    const expectedPatchValue = isChecked;
+  // the expected value is the same as the checked state
+  // (used only for verification if the checkbox has the correct value)
+  const expectedPatchValue = isChecked;
 
-    cy.getCheckboxValue(fieldName, modal).then(theCheckboxValue => {
-      if (isChecked) {
-        if (theCheckboxValue) {
-          // Nothing to do, already checked
-        } else {
-          cy.clickOnCheckBox(fieldName, expectedPatchValue, modal, rewriteUrl, skipRequest);
-        }
+  cy.getCheckboxValue(fieldName, modal).then(theCheckboxValue => {
+    if (isChecked) {
+      if (theCheckboxValue) {
+        // Nothing to do, already checked
       } else {
-        if (theCheckboxValue) {
-          cy.clickOnCheckBox(fieldName, expectedPatchValue, modal, rewriteUrl, skipRequest);
-        } else {
-          // Nothing to do, already unchecked
-        }
+        cy.clickOnCheckBox(fieldName, expectedPatchValue, modal, rewriteUrl, skipRequest);
       }
-    });
+    } else {
+      if (theCheckboxValue) {
+        cy.clickOnCheckBox(fieldName, expectedPatchValue, modal, rewriteUrl, skipRequest);
+      } else {
+        // Nothing to do, already unchecked
```

Comment:
Theeen I guess we don't need that.

Reply:
I though leaving the comment there could help for future readability. do you think i should just take it out?